src/current/cockroachcloud/create-a-serverless-cluster.md

```diff
@@ -47,7 +48,9 @@ After creating a multi-region cluster deployed on AWS, you can optionally [set u
 
 Private connectivity is not available for {{ site.data.products.serverless }} clusters on GCP.
 
-## Step 4. Set resource limits
+Click **Next: Capacity**.
+
+## Step 4. Configure cluster capacity
 
 Your cluster's [resource limits](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#resource-limits) are the maximum amount of storage and RUs you can use in a month. If you reach your storage limit, your cluster will be throttled and you may only be able to delete data. If you reach your RU limit, your cluster will be disabled until the end of the billing cycle unless you raise the limit.
```

Comment:
This section is referencing "resource limits". Glossary may also need to be updated.